adapters/Vungle/VungleAdapter/VungleRouter.m

```diff
@@ -62,7 +62,7 @@ - (void)initWithAppId:(NSString *)appId delegate:(id<VungleDelegate>)delegate {
     return;
   }
 
-  _isInitialising = true;
+  _isInitialising = NO;
```

Comment:
Sorry for the confusion.
Isn't this supposed to be `YES` since Vungle SDK is still initializing?

Reply:
@joshuajplagonera  Yeah, I agree with, it should be YES. Because next step, the SDK will be initialized. Thanks, I will change it to YES. 😄

Reply:
@joshuajplagonera Thanks, done. 😊